documentation/writing_plans.md

```diff
@@ -1,83 +1,91 @@
-# Writing plans in Puppet language
+# Writing plans in the Puppet language
 
-Plans allow you to run more than one task with a single command, compute values for the input to a task, process the results of tasks, or make decisions based on the result of running a task.
+Bolt plans allow you to tie together complex workflows that include multiple
+tasks, scripts, commands, and even other plans.
 
-Write plans in the Puppet language, giving them a `.pp` extension, and place them in the module's `/plans` directory.
+Plans written in the Puppet language allow for more sophisticated control flow
+and better error handling than YAML plans. Puppet plans also allow you to apply
+blocks of Puppet code to remote targets.
 
-Plans can use any combination of [Bolt functions](plan_functions.md#) or [built-in Puppet functions](https://puppet.com/docs/puppet/6.1/function.html).
+When you're writing a plan, you can use any combination of [Bolt 
+functions](plan_functions.md) or [built-in Puppet functions](https://puppet.com/docs/puppet/latest/function.html).
 
-**Related information**  
+> **Note:** For information on how to convert an existing YAML plan to a Puppet
+> plan, see [Converting YAML plans to Puppet language plans](writing_yaml_plans.md).
 
-- [Converting YAML plans to Puppet language plans](writing_yaml_plans.md#)
+## Plan location
 
-## Naming plans
-
-Plan names are based on the filename of the plan, the name of the module containing the plan, and the path to the plan within the module.
+Bolt content follows the same directory structure as Puppet modules. Bolt
+loads plans located in the `site-modules/<MODULE_NAME>/plans` and
```

Comment:
I think including that 'modules/' is for downloaded module content and 'site-modules' is for developing content would be useful here. Something like "Bolt loads downloaded plans from modules/, and local plans from site-modules/". I'm not sure if 'local' or 'custom' is a better word...